### ISSUE TITLE
Disconnection issues and failed p2p on mobile network

### ISSUE THREAD

**Issue by emanuele-f** [ntop/n2n#261]
On a mobile network it's easy to experience p2p failures and supernode connection drops. This should be investigated and fixed. Possibly related to #245 .

**Comment by lucktu** [comment]
Yes, we also found it. we didn't mention it because we thought it had something to do with the peculiarities of the Mobile network, and we were worried that it wouldn't be clear.

**Comment by Logan007** [comment]
Is this still being a widely observed behavior?
